src/framework/components/text/canvas-font.js

```diff
@@ -334,7 +334,7 @@ Object.assign(pc, function () {
             "height": h,
             "xadvance": xadvance / scale,
             "xoffset": xoffset / scale,
```

Comment:
I'm curious why the xoffset value doesn't need to respect the padding like yoffset.